docs/rst/misc.rst

```diff
@@ -0,0 +1,14 @@
+#############
+Miscellaneous
+#############
+
+Inline quotes / citations
+=========================
+
+    This text should be quoted.
+
+Todo items
+==========
+.. todo::
+
+    Foo bar baz.
```

Comment:
Apparently, the `.. todo` directive renders itself with a bit too much of a left padding or margin. Maybe we can do something about it on a future iteration.

![image](https://github.com/crate/crate-docs-theme/assets/453543/54389775-2d30-4b78-8aba-30f2caf23cae)


-- https://crate-docs-theme--378.org.readthedocs.build/en/378/rst/misc.html
-- https://crate-docs-theme--378.org.readthedocs.build/en/378/myst/misc.html

Reply:
Created GH-381 in order to track this item.